src/crate/theme/rtd/conf/__init__.py

```diff
@@ -62,6 +65,24 @@
 version = "latest"
 sitemap_url_scheme = "{lang}{version}{link}"
 
+# Configure intersphinx mapping
+intersphinx_mapping = {
+
+    # CrateDB
+    'reference': ('https://crate.io/docs/crate/reference/', None),
```

Comment:
As @norosa suggested, we should probably link to a version here and at all other occasions.
```suggestion
    'reference': ('https://crate.io/docs/crate/reference/en/latest/', None),
```

Reply:
probably true for some other crate projects too (they vary a bit in terms of what the default version is)

Reply:
I've updated the intersphinx mapping and also added a `links.rst` document in order to verify them.